src/dotnet/Chat.UI.Blazor/Components/ChatFooter.razor

```diff
@@ -111,8 +111,10 @@
 @code {
     [Inject] private Session Session { get; init; } = null!;
     [Inject] private IAccounts Accounts { get; init; } = null!;
+    [Inject] private IAvatars Avatars { get; init; } = null!;
     [Inject] private IChats Chats { get; init; } = null!;
     [Inject] private NavigationManager Nav { get; init; } = null!;
+    [Inject] private ModalUI ModalUI { get; init; } = null!;
```

Comment:
UI services typically go before Nav & the rest ;)

Reply:
done